packages/icon-font/README.md

```diff
@@ -25,7 +25,7 @@ In order to use the icon font, you will need to modify your angular.json file "s
 ```
 "styles": [
     "src/styles.scss",
-    "./node_modules/@brightlayer-ui/icons/iconfont/BrightlayerUIIcons.css"
+    "./node_modules/@brightlayer-ui/icons/packages/icon-font/BrightlayerUIIcons.css"
```

Comment:
These changes are incorrect — the packages folder will never appear in a node_modules folder.

The correct path now will be `./node_modules/@brightlayer-ui/icons/BrightlayerUIIcons.css`.

This will necessitate a new major version on the package because the import path has changed.